packages/blade/docs/guides/Installation.stories.mdx

```diff
@@ -75,16 +75,18 @@ We recommend to use self hosted fonts. We'll use [Fontsource](https://fontsource
 - Import within your app entry file(eg: `App.tsx`, `entryBrowser.tsx` etc.).
   ```js
   import '@fontsource/lato/400.css';
+  import '@fontsource/lato/400-italic.css';
   import '@fontsource/lato/700.css';
+  import '@fontsource/lato/700-italic.css';
```

Comment:
don't we need to update our typography text to accommodate this change?

Reply:
No, it picks it up. Check the implementation of Badge.